File: CB - C/soma_vetor.c
#include <stdio.h>

int main()
{
    int n, i;

    printf("Quantos numeros voce ira digitar? ");
    scanf("%d", &n);

    double vet[n];

    for (i = 0; i < n; i++){
        printf("Digite um numero: ");
        scanf("%lf", &vet[i]);
    }

    printf("\nVALORES: ");
    for (i = 0; i < n; i++){
        printf("%.1lf ", vet[i]);
    }

    double soma = 0;
    for (i = 0; i < n; i++){
        soma = soma + vet[i];
    }

    printf("\nSOMA: %.2lf\n", soma);

    double media = soma / n;
    printf("\nMEDIA: %.1lf\n", media);





    return 0;
}

File: ws - eclipse/soma_vetor.java
import java.util.*;


public class soma_vetor {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		System.out.print("Quantos numeros voce ira digitar? ");
		int n = sc.nextInt();
		
		double[] vet = new double [n];
		
		for(int i = 0; i < n; i++) {
			System.out.print("Digite um numero: ");
			vet[i] = sc.nextDouble();
		}
		
		double soma = 0, media;
		int quanti = 0;
		
		for(int i = 0; i < n; i++) {
			soma = soma + vet[i];
			quanti = quanti + 1;
		}
		
		System.out.print("\nVALORES =  ");
		for(int i = 0; i < n; i++) {
			System.out.print(String.format("%.1f", vet[i]) + " ");
		}
		System.out.println("\nSOMA = " + String.format("%.2f", soma));
		media = soma / quanti;
		System.out.println("MEDIA = " + String.format("%.2f", media));
		
		
		
		
		
		sc.close();
	}

}
